pkgs/development/php-packages/phalcon/default.nix

```
@@ -0,0 +1,27 @@
+{ buildPecl, lib, pcre2, fetchFromGitHub, php, pkg-config }:
+
+buildPecl rec {
+  pname = "phalcon";
+  version = "5.3.0";
+
+  src = fetchFromGitHub {
+    owner = "phalcon";
+    repo = "cphalcon";
+    rev = "v${version}";
+    sha256 = "sha256-82DJ+Qx0OYhw9Nv6FkAoyBec8WWfAiqNfBU9Ll/8RfA=";
```

```suggestion
    hash = "sha256-82DJ+Qx0OYhw9Nv6FkAoyBec8WWfAiqNfBU9Ll/8RfA=";
```

Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.